configuration/openmrs/i18n/reports/locale_fr.json

```diff
@@ -0,0 +1,33 @@
+{
+    "userName": "Nom d'utilisateur",
+    "password": "Mot de passe",
+    "location": "Emplacement",
+    "login": "S'identifier",
+    "selectLocation": "Choisir emplacement",
+    "clinical_key": "Clinique",
```

Comment:
Do you know what those are for?
Maybe we could just remove them.

Reply:
I couldn't find where there are used within the reports app. -I removed those.